src/Ryujinx.HLE/HOS/Services/Nv/NvDrvServices/NvHostAsGpu/NvHostAsGpuDeviceFile.cs

```diff
@@ -111,7 +111,10 @@ private NvInternalResult BindChannel(ref BindChannelArguments arguments)
                 // TODO: Return invalid Fd error.
             }
 
```

Comment:
```suggestion
            else
            {
                channelDeviceFile.Channel.BindMemory(_asContext.Gmm);
            }
```

Or maybe resolve the `TODO` should be enough.